filesystem/replicator_test.go

```diff
@@ -0,0 +1,121 @@
+package filesystem
```

Comment:
Please, add the IBM license header.
You can pick it up from one of the existing code files.